ezbake/config/public/services.d/ca.cfg

```diff
@@ -0,0 +1,5 @@
+# To enable the CA service, leave the following line uncommented
+puppetlabs.services.ca.certificate-authority-service/certificate-authority-service
+# To disable the CA service, comment out the above line and uncomment the line below
+#puppetlabs.services.ca.certificate-authority-disabled-service/certificate-authority-disabled-service
+
```

Comment:
This is super weird that this is showing up in addition to the `bootstrap.cfg` in the public directory above...